src/index.js

```diff
@@ -805,6 +801,14 @@ Bigtable.Cluster = Cluster;
  */
 Bigtable.Instance = Instance;
 
+// Allow creating a `Bigtable` instance without using the `new` keyword.
+/* eslint-disable-next-line no-class-assign */
+Bigtable = new Proxy(Bigtable, {
```

Comment:
That is clever.  @ofrobots does this get us out of jail with the constructor change!?

Reply:
This is super cool!

I am on the fence 🤺though. If we are going to allow non-constructor calling, then we should be consistent about it and support it everywhere. So we ought to do it for `Table` etc. in this library, and in all our other libraries. Many – but not all – of these API classes are rarely constructed, so the performance overhead of the proxy must be considered.

This is a semver major anyway. Let's not try to be cute, as Ryan recently [lamented](https://www.youtube.com/watch?v=M3BM9TB-8yA) us as well.

Reply:
For this library, which is in alpha, I tend to agree.  For others that are GA...  I may take this opt-out approach :)